plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverCommandLine.java

```diff
@@ -65,7 +69,7 @@
 
     public final static Options ALL_OPTIONS = new Options()
         .addOption(PARAM_HELP, false, "Help")
-
+        .addOption(PARAM_CONFIGURATION_FILE, "configuration-file", true, "Use configuration file")
```

Comment:
I think that `configuration-file` is too mouthful. Can we use something shorter?

Also, the description does not describe what this option does.